docs/developer/checkout.mdx

```diff
@@ -980,6 +980,88 @@ A response would look like this:
 }
 ```
 
+
+## Creating order from checkout
+
+:::info
+
+This feature was introduced in **Saleor 3.2**.
+
+:::
+
+:::caution
+
+This feature is currently in **Feature Preview**. This part of Saleor is not complete
+and subject to change but is available to experiment and provide feedback.
+
+:::
+
+The purpose of the operation is to ensure this checkout is correct. The validation consists of checking:
+- The required addresses are valid.
+- The shipping method is valid (if required). 
+- All selected products are in stock.
```

Comment:
The purpose of which operation? I think those are pre-requirements rather than the goal of the operation?